org.okstar.ok-msg.yml

```diff
@@ -0,0 +1,108 @@
+id: org.okstar.ok-msg
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ok-msg-desktop
+rename-icon: ok-msg
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=org.ayatana.indicator.application
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-documents
+  - --unset-env=QT_PLUGIN_PATH
```

Comment:
Why is this needed? Does it ship its own plugins?

Reply:
Resovled it